### PR TITLE
Feat/skeleton view full width

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
Skeleton - full width - expose 'shimmerStyle' prop.

## Changelog
Skeleton - full width - expose 'shimmerStyle' prop.

### CODE REVIEW

File: demo/src/screens/componentScreens/SkeletonViewScreen.tsx [hunk 1]
```diff
@@ -317,6 +318,39 @@ export default class SkeletonViewScreen extends Component {
...
+            showContent={isDataAvailable}
+            renderContent={() => <Button label='Info' size={'small'} link/>}
+          />
+        </View>
+        <SkeletonView
+          shimmerStyle={{width: '100%'/* , height: 30 */}}
+          height={10}
+          borderRadius={5}
+          showContent={isDataAvailable}
+          renderContent={() => <Text>Lorem Ipsum is simply dummy text of the industry.</Text>}
```
Comment (M-i-k-e-l) [inline demo/src/screens/componentScreens/SkeletonViewScreen.tsx:348]: Adding `colors={['red', 'green', 'yellow']}` to the three `SkeletonView`s will show more clearly what I meant when I wrote it's not working properly.

File: src/components/skeletonView/index.tsx [hunk 2]
```diff
@@ -324,7 +330,7 @@ class SkeletonView extends Component<InternalSkeletonViewProps, SkeletonState> {
   };
 
   renderListItemTemplate = () => {
-    const {style, ...others} = this.props;
+    const {style, width, ...others} = this.props; // eslint-disable-line
```
Comment (M-i-k-e-l) [inline src/components/skeletonView/index.tsx:333]: Is this still needed?

Comment (Inbal-Tish) [reply src/components/skeletonView/index.tsx:333]: No. Removing

File: generatedTypes/src/components/skeletonView/index.d.ts [hunk 3]
```diff
@@ -109,9 +109,9 @@ export interface SkeletonViewProps extends AccessibilityProps, MarginModifiers {
      */
     height?: number;
     /**
-     * The width of the skeleton view
+     * The width of the skeleton view (string is not supported for 'circle')
      */
-    width?: number;
```
[the remark this answers is not in the document]
Comment (Inbal-Tish) [reply generatedTypes/src/components/skeletonView/index.d.ts:114]: I can't seem to regenerate the file...

File: demo/src/screens/componentScreens/SkeletonViewScreen.tsx [hunk 4]
```diff
@@ -317,6 +318,40 @@ export default class SkeletonViewScreen extends Component {
...
+          colors={[Colors.red70, Colors.red50, Colors.red70]}
```
[the remark this answers is not in the document]
Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/SkeletonViewScreen.tsx:349]: I did. To remove it?

[the remark this answers is not in the document]
Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/SkeletonViewScreen.tsx:349]: Let's leave it since it a "custom" example

**Review by M-i-k-e-l — APPROVED** [review]

**Pull request merged**